release/build.sh

```diff
@@ -42,6 +42,18 @@ sourcedir="."
 label=""
 timestamp=""
 
+declare -a builds
+function add_build () {
+  builds[${#builds[@]}]=$1
+}
+
+add_build darwin/amd64
+add_build linux/386
+add_build linux/amd64
+add_build windows/386
+add_build windows/amd64
+
```

Comment:
Let's go for 32 bit binaries on linux & windows. Will work for both 32 & 64 bit OSes.